docs/variables/customapi.md

```diff
@@ -8,7 +8,7 @@ Returns the response of a HTTP GET request to a specified URL.
 
 #### Parameters
 
-This variable takes ***one*** *required* parameter that is a URL to send the HTTP GET request to.
+This variable takes **one** *required* parameter that is a **URL** to send the HTTP GET request to.
```

Comment:
is there a lint rule to catch this in the future?

Reply:
Refer to [**this**](https://github.com/fossadev/fossabot-docs/pull/20#discussion_r788759221).